### PR TITLE
Update zh_TW translation with common terms used in Taiwan

### PR DESCRIPTION
**Pull request by hydai**
Explanation:  
I found that translation of zh_TW missed lots of strings and used non-common terms. So I synced the strings with the latest `en/message.json` and updated the Traditional Chinese terms with our common terms.

### CODE REVIEW

**Comment by hydai** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by hydai** [comment]
Hi @darkwing 

I've rebased this branch to latest `develop` branch and all checks have passed.
I would like to know if I need to do anything else to get this PR reviewed. Thanks.

**Comment by andy0130tw** [comment]
Hi @hydai,

I also happened to contribute a localization file update for zh_TW at PR #11212. As the PR becomes stale, I am considering retreating my PR instead of doing rebases. I wonder if we can collaborate on the localization work?

**Comment by hydai** [comment]
Hi @andy0130tw 
That will be great. If we can collaborate on this work.

And I have no idea what's the localization process in MetaMask. Should I create an issue or work item at another place?

**Comment by mcmire** [comment]
Hey, sorry it's taken us so long to get back to you. We have a lot of new translations coming in via #13206, so I'll wait until that's merged first.

**Comment by brad-decker** [comment]
@hydai Thanks for all the work here. I took over and did a rebase. I spot checked the translations to the best of my ability and any where you added translations that our translation service had also added I erred towards our translation service. I removed a few items that were the english versions because our translations system falls back to showing english if a key is missing for a locale. Thanks again, i'll try and get this approved and merged soon!

**Review by brad-decker — DISMISSED** [review]

**Comment by brad-decker** [comment]
Okay cleaned this up again. I'm going to seek a native speaker to check these translations as we now have multiple sources offering different translations. @hydai if you could review the translations and remove any you think are already accurate that would greatly improve our review time.

File: app/_locales/zh_TW/messages.json
Comment (yahsinhuangtw) [inline app/_locales/zh_TW/messages.json:1411]:
```suggestion
    "message": "移除 NFT"
```

**Review by yahsinhuangtw — DISMISSED** [review]
I noticed a few terms were not translated. One additional translation was provided. Overall, the translation sounds natural. Appreciate the work of traditional Mandarin language translation for Taiwan.

**Review by brad-decker — DISMISSED** [review]

**Review by hydai — COMMENTED** [review]
Hi @brad-decker 
I just updated some wording. And the remaining part should be fine. Thanks for the review and assistance.

**Review by brad-decker — DISMISSED** [review]

**Comment by hydai** [comment]
Hi @legobeat 
I just rebased the branch, please let me know if you have any question.

**Review by yahsinhuangtw — APPROVED** [review]
Hello! Thanks for taking the time to work on the zh_TW translation. It's nicely done. Appreciate it.

**Review by legobeat — DISMISSED** [review]

**Comment by hydai** [comment]
Hi @legobeat 
This PR has been stale for a long time. Would you still like to merge it?

**Comment by hydai** [comment]
Hi @legobeat 
I resolved the conflicts. PTAL.

**Pull request closed without merging**
